sdk/purview/Azure.Analytics.Purview.Catalog/Azure.Analytics.Purview.Catalog.sln

```diff
@@ -45,15 +35,15 @@ Global
 		{002A0F15-402B-473D-8842-5507462920BF}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{002A0F15-402B-473D-8842-5507462920BF}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{002A0F15-402B-473D-8842-5507462920BF}.Release|Any CPU.Build.0 = Release|Any CPU
-		{B48EFD57-7500-4A8E-870B-46E3C511B9DF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{B48EFD57-7500-4A8E-870B-46E3C511B9DF}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{B48EFD57-7500-4A8E-870B-46E3C511B9DF}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{B48EFD57-7500-4A8E-870B-46E3C511B9DF}.Release|Any CPU.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {A97F4B90-2591-4689-B1F8-5F21FE6D6CAE}
 	EndGlobalSection
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		..\Azure.Analytics.Purview.Shared\tests\Azure.Analytics.Purview.Shared.Tests.projitems*{002a0f15-402b-473d-8842-5507462920bf}*SharedItemsImports = 5
+		..\Azure.Analytics.Purview.Shared\tests\Azure.Analytics.Purview.Shared.Tests.projitems*{283cb4fe-a207-4d92-8431-6147f1ecbab7}*SharedItemsImports = 13
+	EndGlobalSection
```

Comment:
just curious what these changes are doing?

Reply:
No idea; I removed the stress project and rebuilt via Visual Studio, which regenerated.